src/nvim/autocmd.c

```diff
@@ -112,22 +113,11 @@ static bool autocmd_include_groups = false;
 static char *old_termresponse = NULL;
 
 // Map of autocmd group names and ids.
-//  name -> ID
 //  ID -> name
-static Map(String, int) map_augroup_name_to_id = MAP_INIT;
 static Map(int, String) map_augroup_id_to_name = MAP_INIT;
```

Comment:
I know this is WIP, but just mentioning: we should also be able to eliminate `map_augroup_id_to_name`.